packages/common/base/src/Crossmint/Crossmint.ts

```diff
@@ -0,0 +1,13 @@
+export type CrossmintConfig = {
+    apiKey: string;
+    jwt?: string;
+    overrideBaseUrl?: string;
+};
+
+export class Crossmint {
```

Comment:
Not sure if you intended to do this on another PR, but I would add a public get projectId() that extracts it from the key, as a convenience funciton for debugging / logging if we ask a dev to tell us their project ID. 

Not super critical but seems like 3 LOC to add